### PR TITLE
Release notes 1st August

### PR DESCRIPTION
**Pull request by aapeliv**
+ other tweaks

**Frontend checklist**
- [ ] Formatted my code with `yarn format && yarn lint --fix`
- [ ] There are no warnings from `yarn lint`
- [ ] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ ] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

File: app/frontend/src/features/dashboard/constants.ts [hunk 1]
```diff
@@ -52,21 +52,18 @@ The events are held on Zoom at this link: [https://couchers.org/weekly-social](h
...
-* References: we now send hosts and surfers reminders to write references after hosting experiences
-* Profile: replaced sexes (e.g. female) with genders (e.g. woman) after feedback from the community on the forum
-* Settings: added the user's username on the settings page
-* Admin: implemented a first set of admin tools for moderators and the support team
-* Profile: the avatar upload no longer requires you to scroll down and click "Save" to save the avatar
-* Requests: we fixed a host/surfing request ordering bug which possibly hid some requests for users with more than 10 requests of a given type
-* Signup and settings: we fixed a bug that was preventing users from signing up or changing their password
-* Profile: your visited and lived regions are now sorted
-* Backend: migrated the a large dependency in the backend to a new major version
+* Donations: added a donate page, so you can contribute financially if you'd like to support the project
```
Comment (ItsiW) [inline app/frontend/src/features/dashboard/constants.ts]: add a link

File: app/frontend/src/features/dashboard/constants.ts [hunk 2]
```diff
@@ -52,21 +52,18 @@ The events are held on Zoom at this link: [https://couchers.org/weekly-social](h
...
+* Communities: users are now added to their local communities on signup so the first page load includes this information
+* Contributor: added the contributor back onto the dashboard and on a separate page
```
Comment (ItsiW) [inline app/frontend/src/features/dashboard/constants.ts]: *contributor form

**Comment by aapeliv** [comment]
Waiting for a bit more before merging, but ready for review of the frontend bits.

**Review by darrenvong — DISMISSED** [review]
Frontend looks good to me!

**Pull request merged**